src/lib/framer-motion-features.ts

```diff
@@ -0,0 +1,3 @@
+import { domAnimation } from 'framer-motion'
+
+export default domAnimation
```

Comment:
Are you using this anywhere? Seems like its still imported from 'framer-motion' below